the-collab-lab/tcl-11-smart-shopping-list
# Feature(Issue #7):  Create a welcoming prompt when the shopping list is empty

## Description

This Pull Request shows the user a welcoming prompt that asks them to add an Item by clicking the provided button.

This modification takes place in the `<Listener />` component.


## Related Issue

Closes issue #7

## Acceptance Criteria

- The list view, when there are no items to display, should show a prompt (e.g. a button) for the user to add their first item

## Type of Changes


|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
| ✓   | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/6282435/91347643-acf1a780-e7b0-11ea-9937-8bdbd02b85ae.png)


### After

![image](https://user-images.githubusercontent.com/6282435/91346816-60599c80-e7af-11ea-8de3-6aaa9b482b82.png)


## Testing Steps / QA Criteria

- From your terminal, run git checkout `bl-kl-welcome-prompt` branch
- Once once checked out to the `bl-kl-welcome-prompt` branch, in the browser, visit the new page: (http://localhost:3000 or as indicated in your terminal).
- Click the `List` link in the `Footer` below.
- You should see a `<Card />` component that informs you that your shopping list is empty, and ask for you to add an item.
